### PR TITLE
Remove jcenter and add Typescript

### PR DESCRIPTION
**Pull request by JonasWho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,12 +30,8 @@
...
   "licenseFilename": "LICENSE",
   "readmeFilename": "README.md",
-  "peerDependencies": {
-    "react": "^16.8.1",
-    "react-native": ">=0.59.0-rc.0 <1.0.x"
-  },
   "devDependencies": {
-    "react": "^16.8.3",
-    "react-native": "^0.59.10"
+    "@types/react-native": "^0.66.11",
```
Comment (martinpoulsen) [inline package.json:34]: Is there a reason to not go for a newer version of RN types?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ _The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
...
 ## [unreleased]
 
+### Fixed
+
+- Removed the use of jcenter for Android
+
+### Added
+
+- Typescript support
+
```
Comment (martinpoulsen) [inline CHANGELOG.md:16]: We should mention here that the Android native SDK is updated to v. 4.1.0

**Review by martinpoulsen — APPROVED** [review]
Looks good! Great to see this lib converted to TypeScript 🥳

File: package.json (hunk 1 above)
Comment (JonasWho) [reply package.json]: Good point, thanks!

**Pull request merged**